indra_network_search/frontend/generate_css_file.py

```diff
@@ -0,0 +1,11 @@
+import logging
+from pathlib import Path
+from indra.assemblers.html.assembler import generate_source_css
+
+logger = logging.getLogger(__name__)
+
+CSS_PATH = Path('./src/assets/sources.css').absolute().as_posix()
```

Comment:
would suggest doing this relative to current file location with `Path(__file__)`